octopusdeploy/schema_project.go

```diff
@@ -139,9 +136,16 @@ func flattenProject(project *octopusdeploy.Project) map[string]interface{} {
 		"template":                             flattenActionTemplateParameters(project.Templates),
 		"tenanted_deployment_participation":    project.TenantedDeploymentMode,
 		"variable_set_id":                      project.VariableSetID,
-		"version_control_settings":             flattenVersionControlSettings(project.VersionControlSettings),
 		"versioning_strategy":                  flattenVersioningStrategy(project.VersioningStrategy),
 	}
+
+	if project.PersistenceSettings != nil {
+		if project.PersistenceSettings.GetType() != "Database" {
```

Comment:
this might be safer as `== "VersionControlled"`. I.e. look for the single true case rather than assume there will only ever be a single false case. It is unlikely another type will get added, but such assumptions have caught us out in the past.